docs/configuration/unit_testing_rules.md

```diff
@@ -25,6 +25,11 @@ rule_files:
 # optional, default = 1m
 evaluation_interval: <duration>
 
+# optional, default = 0m
+# By default the time is mocked and starts from 0 timestamp.
```

Comment:
No need to mention mocking, as among other things we don't actually mock and if we did that'd be an implementation detail.